src/Client.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright 2023-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace MongoDB\Bundle;
+
+use MongoDB\Client as MongoDBClient;
+use MongoDB\Driver\Monitoring\Subscriber;
+
+final class Client extends MongoDBClient
+{
+    /**
+     * @internal
+     */
+    public function addSubscriber(Subscriber $subscriber): void
```

Comment:
This should be added to MongoDB\Client, so that we can remove this class. PHPLIB-1309

Reply:
Fixed by using a configurator.